solar/solar/events/controls.py

```diff
@@ -91,15 +94,15 @@ class React(Event):
 
     def insert(self, changed_resources, changes_graph):
 
-        if self.parent in changes_graph:
-            if self.dependent not in changes_graph:
+        if self.parent_node in changes_graph:
+            if self.child_node not in changes_graph:
                 changes_graph.add_node(
-                    self.dependent, status='PENDING',
+                    self.child_node, status='PENDING',
```

Comment:
i changed it already in separate request, will rebase that request after this one will be merged